packages/components/scripts/generateDSRComponents.ts

```diff
@@ -49,10 +49,11 @@ const generateDSRComponents = (): void => {
         .replace(/\n  public componentDidRender\(\): void {[\s\S]+?\n  }\n/g, '')
         .replace(/\n  public disconnectedCallback\(\): void {[\s\S]+?\n  }\n/g, '')
         .replace(/\n  public componentShouldUpdate\([\s\S]+?\n  }\n/g, '')
-        .replace(/\n  private (?!get).*{[\s\S]+?\n  };?\n/g, '') // private methods without getters
+        .replace(/\n  private (?!get).+(\n.+)?\{[\s\S]+?\n  };?\n/g, '') // private methods without getters
         .replace(/\nconst propTypes[\s\S]*?};\n/g, '') // temporary
         .replace(/\s+validateProps\(this, propTypes\);/, '')
         .replace(/\s+attachComponentCss\([\s\S]+?\);/, '')
+        .replace(/\s+this\.validateInitialValue\(\);/, '')
```

Comment:
Isn't this very component specific? I guess we shouldn't try to replace this for every component.